crates/red_knot_workspace/src/workspace.rs

```diff
@@ -105,8 +107,9 @@ impl Workspace {
             packages.insert(package.root.clone(), Package::from_metadata(db, package));
         }
 
-        Workspace::builder(metadata.root, None, packages)
+        Workspace::builder(metadata.root, packages)
             .durability(Durability::MEDIUM)
+            .open_file_set_durability(Durability::LOW)
```

Comment:
nit: I know it's not new to this PR, but I'd consider renaming the attribute on `Workspace` to be `open_fileset` rather than `open_file_set`. Looking throught this PR, I assumed `open_file_set_durability` set the durability for a single open file, rather than setting the durability for the set of currently open files. "Fileset" is a word, and I think it would be just as grammatical to refer to an "open fileset" as it would be to refer to an "open file set".